README.md

```diff
@@ -57,16 +57,15 @@ NFD is used to label nodes with the following labels:
 - RDMA capability
 - GPU features*
 
+>__NOTE__: We use nodeFeatureRules to label PCI vendor and device and it's enabled by default using `nfd.deployNodeFeatureRules` flag
```

Comment:
nit: nodeFeatureRules should be a link.

Reply:
also its unclear in the context of this section what is nfd.deployNodeFeatureRules flag.

im not even sure helm values are regarded as "flags"

Reply:
chart parameter